OptiFineDoc/assets/minecraft/optifine/lang/ru_ru.lang

```diff
@@ -137,7 +138,7 @@ options.guiScale.tooltip.2=  Авто - максимальный размер
 options.guiScale.tooltip.3=  Мелкий, обычный, крупный - 1x—3x соответственно
 options.guiScale.tooltip.4=  с 4x до 10x - на мониторах с разрешением 4K
 options.guiScale.tooltip.5=Нечётные значения несовместимы со шрифтом Unicode.
-options.guiScale.tooltip.6=Интерфейс может быть быстрее при меньших значениях.
+options.guiScale.tooltip.6=Производительность может быть выше при меньших значениях.
```

Comment:
The string doesn't fit the tooltip box using the default font. I suggest you to replace it with:
```suggestion
options.guiScale.tooltip.6=Производительность может повысится при уменьшении
options.guiScale.tooltip.7=значения параметра.
```
Before:
![image](https://user-images.githubusercontent.com/17567951/127075496-1906f095-e1ab-4086-a205-0bbf87c35f1d.png)

After:
![image](https://user-images.githubusercontent.com/17567951/127075447-11001c08-cfd2-4712-bdb1-d6ff192ff534.png)